### DISCUSSION TITLE
How to disable SSL verification for version 4.4?

### DISCUSSION THREAD

**Discussion by mhasanen** [Ginger-Automation/Ginger#2652] [Q&A]
How to disable SSL verification for version 4.4? all our api requests stopped to work after installing v4.4.  When running the same request with postman it works if the SSL verification is disabled.
![2022-09-06 09_00_09-Ginger by amdocs](https://user-images.githubusercontent.com/110529960/188568026-c6eb06ee-6936-428a-b344-a752c8b26c7a.png)

**Comment by digvijayAMDX** [comment]
Please check the SSL Settings in the operation settings tab of the action where you are making api call for ssl and try with different values of Security Types as shown in the below screenshot. Please reach out to use if still have an issue.
![REST_SSL_Settings](https://user-images.githubusercontent.com/59822903/191000791-290268d2-f1a0-4190-99cd-5819ffcb0905.png)
